src/utils/react.js

```diff
@@ -14,17 +15,8 @@ export let wrapDisplayName = (name, Component) => Wrapped => {
   return Wrapped
 }
 
-// these are for futil :)
-// (x -> y) -> string -> {x} -> y
-let getWith = _.curry((customizer, path, object) =>
-  customizer(_.get(path, object))
-)
-// ({a} -> {b}) -> {a} -> {a, b}
-let expandObject = _.curry((transform, obj) => ({ ...obj, ...transform(obj) }))
-
-// (a -> {b}, k) -> Component<{k: a}> -> Component<{b}>
-export let lensify = _.flow(
-  getWith,
-  expandObject,
+// (k, a -> {b}) -> Component<{k: a}> -> Component<{b}>
+export let expandProp = _.flow(
+  expandObjectBy,
```

Comment:
@stellarhoof FYI, you'll like this much more :)

Reply:
This reads better for sure. I still don't like `getWith` though